CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contribution guidelines
+
+Thank you for your interest in contributing. It is greatly appreciated.
+
+# Pull requests
+
+Code contributions to this project follow this process and set of requirements:
+
+1. As a contributor you should create an issue describing the suggested change on [our Redmine installation platform.dandigbib.org](http://platform.dandigbib.org/) in Danish. This allows the business side of the project to evaluate your suggestion. If it accepted for inclusion it will be marked for review. The only exception from this rule is if the change does not affect end users in any way. This could be developer documentation changes, technical tests or changes to tools.
```

Comment:
As a contributor you should create an issue describing the suggested change on http://platform.dandigbib.org/projects/ddb-cms/issues/new (Danish). To create an issue you must create an account via http://platform.dandigbib.org/account/register.